sdk/playwrighttesting/microsoft-playwright-testing/src/utils/reporterUtils.ts

```diff
@@ -318,8 +318,10 @@ class ReporterUtils {
   };
 
   public static isTimeGreaterThanCurrentPlus10Minutes(isoString: string): boolean {
+    // Convert UTC to local
+    const localTimeString = new Date(isoString).toLocaleString();
```

Comment:
why is this required? `this.sasUri.expiresAt` should be in UTC format and new Date(isostring) would return UTC timestamp

Reply:
The problem is at Date.parse(), it expected local time unless mentioned in date string. In our case, this.sasUri.expiresAt is indeed UTC but it doesnt mention any timezone, hence Datw.parse() assume it to be local timezone, hence calculate wrong timestamp. 
We could solve it differently as well like instead of relying on this.sasUri.expiresAt , we could actually fetch expiry from this.sasUri.uri, that would be more accurate.